src/main/java/uk/gov/hmcts/reform/sscs/controller/SyaController.java

```diff
@@ -43,4 +47,20 @@
 
         return status(201).build();
     }
+
+
+    @ApiOperation(value = "submitDraftAppeal", notes = "Creates a draft appeal", response = Draft.class)
+    @ApiResponses(value =
+            {@ApiResponse(code = 201, message = "Submitted draft appeal successfully", response = Draft.class)})
+    @PostMapping(value = "/drafts", consumes = APPLICATION_JSON_VALUE)
+    public ResponseEntity<Draft> createDraftAppeal(
+            @RequestHeader(AUTHORIZATION) String authorisation,
+            @RequestBody SyaCaseWrapper syaCaseWrapper) {
+        Preconditions.checkNotNull(syaCaseWrapper);
+        Draft draft = Draft.builder()
+                .id(submitAppealService.submitDraftAppeal(authorisation, syaCaseWrapper))
+                .build();
+        log.info("{} processed successfully", draft);
+        return status(HttpStatus.CREATED).body(draft);
```

Comment:
Should this not also return the http header `location`?

Reply:
done